### PR TITLE
[SASS-7967] Create SectionCompletedState page

### PR DESCRIPTION
**Pull request by Jake-Raffe**
# Checklist

Jira subtask: SASS-____

## Developer: Before Asking Others To Review Your Code

Have you ...

- [ ]  Had a QA verify that all acceptance criteria have been met, with comment(s) in the ticket? 
- [ ]  Hosted a group review of your implementation?
- [ ]  Rebased against `main`? 
- [ ]  Ensured that your commit(s) contain the ticket key?
- [ ]  Ensured that your branch is named after the ticket key?
- [ ]  Successfully run the unit tests?
- [ ]  Successfully run the integration tests?
- [ ]  Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.
- [ ]  Ensured that the PR job has passed.

## Reviewer: Before Reviewing

Have you ...

- [ ]  Pulled the branch down?
- [ ]  Assigned yourself to this PR?
- [ ]  Moved the Jira *subtask* to “in review”?

## Developer: After Approval

Have you ...

- [ ]  Moved the Jira *subtask* to “in pipeline”?

## Developer: After Merging

Have you ...

- [ ]  Scheduled a demo? If not, please add a comment.

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -31,6 +31,9 @@ GET         /:taxYear/pensions-summary
 
 GET         /:taxYear/overseas-pensions                                                                                            controllers.pensions.OverseasPensionsSummaryController.show(taxYear: Int)
 
+GET         /:taxYear/:journey/section-completed                                                                                   controllers.pensions.SectionCompletedStateController.show(taxYear: Int, journey: String)
```
Comment (harry-ritchie) [inline conf/app.routes:34]: Was the thinking to have `controllers...(..., journey: **Journey**)`? I assume this was the intent hence the `PathBindable`?

File: app/models/mongo/JourneyStatus.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.mongo
+
+import enumeratum._
```
Comment (harry-ritchie) [inline app/models/mongo/JourneyStatus.scala:19]: Just out-of-interest, what was the thinking behind pulling in `enumeratum`, and not using scala built-in "support" for enums (Sealed trait & cc/co?). Does enuermatum have easier/better integration with PathBindable?

File: conf/app.routes (hunk 1 above)
Comment (Jake-Raffe) [reply conf/app.routes:34]: Yep, changed it and forgot to change it back! Thanks

File: app/models/mongo/JourneyStatus.scala (hunk 2 above)
Comment (Jake-Raffe) [reply app/models/mongo/JourneyStatus.scala]: Have removed enumaratum, you were right it was a lot of unnecessary extra boilerplate code

**Pull request merged**